registry.k8s.io/images/k8s-staging-etcd/images.yaml

```
@@ -20,6 +20,7 @@
     "sha256:e013d0d5e4e25d00c61a7ff839927a1f36479678f11e49502b53a5e0b14f10c3": ["3.5.9-0"]
     "sha256:22f892d7672adc0b9c86df67792afdb8b2dc08880f49f669eaaa59c47d7908c2": ["3.5.10-0"]
     "sha256:29901446ff08461789b7cd8565fc5b538134e58f81ca1f50fd65d0371cf6571e": ["3.5.11-0"]
+    "sha256:44a8e24dcbba3470ee1fee21d5e88d128c936e9b55d4bc51fbef8086f8ed123b": ["3.5.11-0"]
```

This should be `3.5.12-0`, right?
```suggestion
    "sha256:44a8e24dcbba3470ee1fee21d5e88d128c936e9b55d4bc51fbef8086f8ed123b": ["3.5.12-0"]
```

---

Yes, thank you very much.